### DISCUSSION TITLE
Помогите с настройкой

### DISCUSSION THREAD

**Discussion by xLuxuT** [masterking32/MasterHamsterKombatBot#92] [Q&A] [answered]
Помогите разобраться с настройкой. Что я не так делаю? Покупает все подряд карточки и очень не выгодные.

             auto_tap": True,  # Enable auto tap by setting it to True, or set it to False to disable
            "auto_free_tap_boost": True,  # Enable auto free tap boost by setting it to True, or set it to False to disable
            "auto_get_daily_cipher": True,  # Enable auto get daily cipher by setting it to True, or set it to False to disable
            "auto_get_daily_task": True,  # Enable auto get daily task by setting it to True, or set it to False to disable
            "auto_get_task": True,  # Enable auto get (Youtube/Twitter and ...) task to True, or set it to False to disable
            "auto_finish_mini_game": True,  # Enable auto finish mini game by setting it to True, or set it to False to disable
            "auto_playground_games": True,  # Enable auto playground games by setting it to True, or set it to False to disable
            # If you have more than 5 accounts, you need to disable the auto_playground_games feature or use a proxy for each account.
            "auto_upgrade": True,  # Enable auto upgrade by setting it to True, or set it to False to disable
            "auto_upgrade_start": 350_000_000,  # Start buying upgrades when the balance is greater than this amount
            "auto_upgrade_min": 300_000_000,  # Stop buying upgrades when the balance is less than this amount
            # This feature will ignore the auto_upgrade_start and auto_upgrade_min.
            # By changing it to True, the bot will first find the overall best card and then wait for the best card to be available (based on cooldown or price).
            # When the best card is available, the bot will buy it and then wait for the next best card to be available.
            # This feature will stop buying upgrades when the balance is less than the price of the best card.
            "wait_for_best_card": False,  # Recommended to keep it True for high level accounts
            "enable_parallel_upgrades": True,  # Enable parallel card upgrades. This will buy cards in parallel if the best card is on cooldown. It should speed up the profit.
            "parallel_upgrades_max_price_per_hour": 1_000_000,  # Cards with less than X coins per 1k will be bought

**Comment by masterking32** [comment, marked as answer]
```
        "wait_for_best_card": True,  # Recommended to keep it True for high level accounts
        "enable_parallel_upgrades": True,  # Enable parallel card upgrades. This will buy cards in parallel if the best card is on cooldown. It should speed up the profit.
        "parallel_upgrades_max_price_per_hour": 1000,  # Cards with less than X coins per 1k will be bought
```
